lib/index.ts

```diff
@@ -1,6 +1,7 @@
 // NOTE: Aliases to classes and interfaces are used for external consumption.
 
 // SDK exports.
+import ISigner from './interfaces/ISigner';
```

Comment:
Don't we want to export LocalSigner here as well? So people won't have to reimplement it.

Reply:
The intention was actually not to expose this class, but good point that I still need to expose the static factory method!!

I will expose it and not over think it!